apim-operator/pkg/controller/api/api_controller.go

```diff
@@ -514,36 +513,36 @@ func (r *ReconcileAPI) Reconcile(request reconcile.Request) (reconcile.Result, e
 					certList[alias] = certPath + defaultCert.Name + "/" + certName
 					defaultSecConf.CertificateAlias = alias
 				}
-			//copying default security to user namespace
-			log.Info("copying default security to " + userNameSpace)
-			newDefaultSecurity := copyDefaultSecurity(securityDefault, userNameSpace, owner)
-			errCreateSecurity := r.client.Create(context.TODO(), newDefaultSecurity)
-			if errCreateSecurity != nil {
-				log.Error(errCreateSecurity, "error creating secret for default security in user namespace")
-				return reconcile.Result{}, errCreateSecurity
-			}
-			log.Info("default security successfully copied to " + userNameSpace + " namespace")
-			if newDefaultSecurity.Spec.SecurityConfig[0].Issuer != "" {
-				defaultSecConf.Issuer = newDefaultSecurity.Spec.SecurityConfig[0].Issuer
-			}
-			if newDefaultSecurity.Spec.SecurityConfig[0].Audience != "" {
-				defaultSecConf.Audience = newDefaultSecurity.Spec.SecurityConfig[0].Audience
-			}
-			defaultSecConf.ValidateSubscription = newDefaultSecurity.Spec.SecurityConfig[0].ValidateSubscription
-		} else if errGetSec != nil {
-			log.Error(errGetSec, "error getting default security from user namespace")
-			return reconcile.Result{}, errGetSec
-		} else {
-			log.Info("default security exists in " + userNameSpace + " namespace")
-			//check default cert exist in usernamespace
-			var defaultCertUsrNs = &corev1.Secret{}
-			errCertUserns := r.client.Get(context.TODO(), types.NamespacedName{Name: securityDefault.Spec.SecurityConfig[0].Certificate, Namespace: userNameSpace}, defaultCertUsrNs)
-			if errCertUserns != nil && errors.IsNotFound(errCertUserns) {
-				log.Error(errCertUserns, "default certificate is not found in user namespace")
-				return reconcile.Result{}, errCertUserns
-			} else if errCertUserns != nil {
-				log.Error(errCertUserns, "error retrieving default certificate in user namespace")
-				return reconcile.Result{}, errCertUserns
+				//copying default security to user namespace
+				log.Info("copying default security to " + userNameSpace)
+				newDefaultSecurity := copyDefaultSecurity(securityDefault, userNameSpace, owner)
+				errCreateSecurity := r.client.Create(context.TODO(), newDefaultSecurity)
+				if errCreateSecurity != nil {
+					log.Error(errCreateSecurity, "error creating secret for default security in user namespace")
+					return reconcile.Result{}, errCreateSecurity
+				}
+				log.Info("default security successfully copied to " + userNameSpace + " namespace")
+				if newDefaultSecurity.Spec.SecurityConfig[0].Issuer != "" {
```

Comment:
Can SecurityConfig[0] be null? If that's the case, we may need handle this.

Reply:
Updated the PR with reverting corrected formatting errors in other codes. Hence force pushed with only my changes.